backend/app/models/image.py

```
@@ -0,0 +1,40 @@
+import uuid
+from datetime import datetime
+from typing import Optional, TYPE_CHECKING, List, Any
```

Ruff (0.11.9)

`typing.List` is deprecated, use `list` instead

(UP035)